cardano-api/src/Cardano/Api/Typed/Internal.hs

```diff
@@ -0,0 +1,12 @@
+module Cardano.Api.Typed.Internal
+  ( toNetworkMagic
+  ) where
```

Comment:
I think we should probably just expose the `NetworkMagic` type from the API. It's a reasonable concept to expose.

So we don't need to prematurely start splitting things out of the main API module in an ad-hoc way. Yes we want to break it up properly now that we've retired the old API, but we want to do that in a coherent way and all at once.

Reply:
Fixed!